### DISCUSSION TITLE
Single series not working

### DISCUSSION THREAD

**Discussion by jasonlshelton** [Flexget/Flexget#3012] [Q&A]
I've had my setup working for a while now. I use thetvdb as a source and deluge as my torrent software. I've struggled with configs over time, especially with updates to flexget, but I haven't changed anything in a while.

I have one show (The flash, if that matters) that refuses to download. Flexget queries thetvdb and sees the series, I get this in the log
`2021-05-20 12:30:11 VERBOSE  api_tvdb      tv_shows        Data for The Flash (2014) has expired, refreshing from tvdb`

And nothing else. I know my torrent source has the recent shows because I've been getting them manually. ANy ideas why flexget doesn't like this one? Is there a way to delete the whole series then scan it back into the flexget database?

Any ideas welcome

**Comment by ksurl** [comment]
Could be a tvdb issue. Other series lookup via tvdb just fine? If you want to forget just this series you can run `flexget series forget "The Flash"`
